src/nl/surf/eduhub_rio_mapper/cli.clj

```diff
@@ -195,12 +222,12 @@
         {:keys [getter resolver ooapi-loader]
          :as   handlers}             (make-handlers config)
         queues                       (clients-info/institution-schac-homes clients)
+        {:keys [cmd delete-by-code]} (if (= "delete-by-code" command) {:delete-by-code true :cmd "delete"} {:cmd command})
```

Comment:
This is ugly and a weird preparation to do for something >20 lines down.  Please keep the uglyness of this local to the item in the case-statement below.